lib/audited/audit.rb

```diff
@@ -55,6 +55,10 @@ class Audit < ::ActiveRecord::Base
       serialize :audited_changes, YAMLIfTextColumnType
     end
 
+    if Rails.gem_version >= Gem::Version.new("7.0") && Audited.encrypt_audited_changes
```

Comment:
This doesn't seem to work as expected, it appears that this is evaluated before the `Audited.encrypt_audited_changes = true` in the initializer runs so it evaluates to false initially and is never re-evaluated. The `encrypts` call below never runs as a result.